### PR TITLE
Final scene fix

### PR DESCRIPTION
**Pull request by Ittis**
Fixed final scene and added postFX to skined pipeline

### CODE REVIEW

File: Game/Resources/final_event/MainScript.as [hunk 1]
```diff
@@ -61,7 +61,7 @@ class MainScript {
         switch (phase) {
             case 0: { // Entering final scene.
                 vec3 pos = minecart.GetWorldPosition();
-                pos.x += speed * deltaTime;
+                pos.x -= speed * deltaTime;
```
Comment (andens) [inline Game/Resources/final_event/MainScript.as]: I don't see that the scene has been rotated in the superscene? Is this a leftover from testing locally?

File: Game/Resources/final_event/MainScript.as [hunk 2]
```diff
@@ -94,7 +93,7 @@ class MainScript {
             }
             case 3: { // Continue after monster has been killed.
                 vec3 pos = minecart.GetWorldPosition();
-                pos.x += speed * deltaTime;
+                pos.x -= speed * deltaTime;
```
Comment (andens) [inline Game/Resources/final_event/MainScript.as]: Leftover?

File: Game/Resources/final_event/MainScript.as [hunk 3]
```diff
@@ -113,7 +112,7 @@ class MainScript {
 
                 if (fogTimer >= fadeStartTime) {
                     fadeTimer += deltaTime;
-                    float fadeRatio = fadeTimer / fadeApexDuration;
+                    float fadeRatio = fadeTimer / fogApexDuration;
```
Comment (andens) [inline Game/Resources/final_event/MainScript.as]: This should still be `fadeApexDuration`. They apply to different techniques that are interleaved to create the final fade effect.

File: Game/Resources/final_event/MainScript.as (hunk 1 above)
Comment (Ittis) [reply Game/Resources/final_event/MainScript.as]: No this is needed in superscene

File: Game/Resources/final_event/MainScript.as (hunk 2 above)
Comment (Ittis) [reply Game/Resources/final_event/MainScript.as]: Nope

File: Game/Resources/final_event/MainScript.as (hunk 3 above)
Comment (Ittis) [reply Game/Resources/final_event/MainScript.as]: whoops

**Comment by Ittis** [comment]
iamheman

**Pull request merged**